ouroboros-network-framework/ouroboros-network-framework.cabal

```diff
@@ -115,6 +115,37 @@ library
                        -Wredundant-constraints
                        -Wno-unticked-promoted-constructors
 
+library testlib
+    visibility: public
+    hs-source-dirs: testlib
+
+    exposed-modules: TestLib.ConnectionManager
+                     TestLib.InboundGovernor
+                     TestLib.Utils
+
+    other-modules:
+
+    build-depends:   base
+                   , bytestring
+                   , containers
+
+                   , QuickCheck
+
+                   , io-sim
+                   , ouroboros-network-framework
+
+  default-language:  Haskell2010
+  ghc-options:       -rtsopts
+                     -threaded
+                     -Wall
+                     -Wcompat
+                     -Wincomplete-uni-patterns
+                     -Wincomplete-record-updates
+                     -Wpartial-fields
+                     -Widentities
+                     -Wredundant-constraints
+                     -Wno-unticked-promoted-constructors
```

Comment:
This one is probably not needed, unless we use `DataKinds` somewhere.

Reply:
done